test/realtime/live_objects.test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+define(['ably', 'shared_helper', 'async', 'chai', 'live_objects'], function (
+  Ably,
+  Helper,
+  async,
+  chai,
+  LiveObjectsPlugin,
+) {
+  var expect = chai.expect;
+  var createPM = Ably.protocolMessageFromDeserialized;
+
+  function LiveObjectsRealtime(helper, options) {
+    return helper.AblyRealtime({ ...options, plugins: { LiveObjects: LiveObjectsPlugin } });
+  }
+
+  async function monitorConnectionThenCloseAndFinish(helper, action, realtime, states) {
```

Comment:
ok that's fair, but it's a generic helper so ought to be part of the shared helpers. also seems unnecessary taking `helper` as an argument rather than being a method on the helper class, like `helper.monitorConnectionThen...`

Reply:
true, now that's this function is being used in multiple tests it make sense to move it to the helper. will do that

Reply:
moved `monitorConnectionThenCloseAndFinish` to the helper in a separate commit: https://github.com/ably/ably-js/pull/1880/commits/fac8a6412351433ffa1bf70d69106205118dd2f6